web/src/main/webapp/WEB-INF/views/anatomyFrag.jsp

```diff
@@ -48,7 +48,7 @@
 					            <td>${row.phenotypingCenter} </td>
 					            <td>${row.numberOfImages} </td>
 					            <td>
-					            	<c:if test="${row.getEvidenceLink().getDisplay()}">
+					            	<c:if test="${row.numberOfImages > 0}">
```

Comment:
Using evidenceLink().getDisplay() is consistent with the other tables using DataTableRow (gene,phenotype,experiment). I think it would be nicer to have the same test, but I recognize the custom one works too.